### PR TITLE
[WIP] Minor updates for remote conference

### PR DESCRIPTION
**Pull request by nsreeen**

### CODE REVIEW

File: _pages/coc.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ Contact information
 
 During the conference we will have dedicated people who will be available to talk about anything relating to the code of conduct or attendees comfort and well being.
 
-Please email us at [coc@enthusiasticon.de](mailto:coc@enthusiasticon.de) if you have any concerns about this Code of Conduct. You can also email us during the conference or use the following form (anonymous, via [formspree.io](https://formspree.io)):
+Please email us at [coc@enthusiasticon.de](mailto:coc@enthusiasticon.de) if you have any concerns about this Code of Conduct.  
+
+If you would like to contact us anonymously, you can use the following form (via formspree.io):
```
Comment (nsreeen) [inline _pages/coc.md:36]: Don't think we should remove the form incase anyone wants to get in touch anonymously.  Maybe we should say that we might not respond to emails during the conference very quickly though

Comment (Dzol) [reply _pages/coc.md:36]: > Don't think we should remove the form incase anyone wants to get in touch anonymously

Agreed.

> Maybe we should say that we might not respond to emails during the conference very quickly though

How about saying that we'll do our best to respond ASAP?

Comment (zormit) [reply _pages/coc.md:36]: The last time we had two "Code of Conduct people", who watched the email inbox closely and were also present at the conference.

**Comment by clample** [comment]
The wording fits the remote conference much nicer now - thanks @nsreeen

**Pull request merged**